support-metrics-common/src/main/java/io/confluent/support/metrics/BaseSupportConfig.java

```diff
@@ -0,0 +1,324 @@
+/**
```

Comment:
moved from client ( didn't move complete file , had to leave few logic in clients so that this is generic)